app/mobile-app/src/components/LandingScreen.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
```

Comment:
I think it might be worth setting up [Native testing library](https://callstack.github.io/react-native-testing-library/docs/getting-started) with this. Seems to offer some much nicer API on top of `react-test-renderer`, and encourages similar testing ideas as React Testing Library that we're using for the web frontend, which I think is good to follow here too. What do you think?

Reply:
Good suggestion, I'll look at it.